src/getErrors.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-len */
+const getErrors = (text, toCase) => {
+  const correctCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!toCase) {
+    errors.push({
+      message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  } else if (!(correctCases.includes(toCase))) {
```

Comment:
You don't need else if here, bc if !toCase you will have toCase )

```suggestion
  }
  
  if (!(correctCases.includes(toCase))) {
```